Cargo.lock

```diff
@@ -5457,9 +5457,9 @@ dependencies = [
 
 [[package]]
 name = "solana-frozen-abi"
-version = "1.14.8"
+version = "1.15.1"
```

Comment:
This is a bit strange and might explain the recompilations... why are we picking up version 1.15 when spl-token-2022 is bound to 1.14?